addons/stock/report/report_stock_forecasted.py

```diff
@@ -143,7 +143,7 @@ def _get_report_lines(self, product_template_ids, product_variant_ids, wh_locati
         ins_per_product = defaultdict(lambda: [])
         for in_ in ins:
             ins_per_product[in_.product_id.id].append([in_.product_qty, in_])
-        currents = {c['id']: c['qty_available'] for c in outs.product_id.read(['qty_available'])}
+        currents = {c['id']: c['qty_available'] for c in outs.product_id.with_context(location=wh_location_ids).read(['qty_available'])}
```

Comment:
I'm not sure to understand why you need to do that ?
I though the `warehouse` put in the context in the `_get_report_data` was already enough to address to this commit's issue ?

Reply:
It is not for the report it-self. But for the computation of forecasted information on the stock move which call `_get_report_lines` without set a warehouse in context.